src/dra/5D5BC.c

```diff
@@ -1515,19 +1515,19 @@ void InitStatsAndGear(bool isDeathTakingItems) {
 void DrawRichterHud(void) {
     Primitive* prim;
 
-    D_80137978 = 400;
-    D_8013797C = 400;
+    g_PlayerHud.unk0C = 400;
+    g_PlayerHud.unk10 = 400;
     D_801397FC = 0;
     D_80139008 = 0;
-    D_80137994 = 0;
+    g_PlayerHud.unk28 = 0;
     D_8003C744 = 0;
-    D_80137980 = 48;
-    D_80137984 = 0;
-    D_80137990.unk0 = 0;
-    D_8013798C = 40000 / D_80137978;
-    D_80137988 = 40000 / D_8013797C;
-    D_80137970 = func_800EDD9C(PRIM_GT4, 9);
-    prim = &g_PrimBuf[D_80137970];
+    g_PlayerHud.unk14 = 48;
+    g_PlayerHud.unk18 = 0;
+    g_PlayerHud.unk24 = 0;
+    g_PlayerHud.unk20 = 40000 / (u32)g_PlayerHud.unk0C;
```

Comment:
unk0C is already u32 so I think this can be removed?
```
 /* 80072F2C */ u32 unk0C;
```

Reply:
Good point, that struct was previously all s32 so the cast was needed.

Reply:
wait no, unk0C is in PlayerHud, your quoted line is in g_Player.unk0C.

In PlayerHud, it needs to be an s32 due to the `if (g_PlayerHud.unk0C < 0) {` at line 1673 of this file. So it must be s32, and must have a cast at this usage.

Reply:
I misread, no problem then